src/remixAppManager.js

```diff
@@ -213,6 +213,21 @@ export class RemixAppManager extends PluginEngine {
       version: '0.1.4-beta',
       location: 'sidePanel'
     }
+    const lexonCompiler = {
```

Comment:
this is not necessary as the declaration is already in `remix-plugins-directory` repo.
Could you also rebase this PR because we did some changes recently in this file.